### ISSUE TITLE
Players hear different songs playing.

### ISSUE THREAD

**Issue by FlagHater** [Batres3/BoomboxPlaylists#1]
All the players hear different custom songs we've added, but no one can hear the same, synchronized, music.

**Comment by Batres3** [comment]
Is this with the latest update on thunderstore? it's farther ahead than this github, which I will update when I have time. If you are using the Github version then it'll probably get fixed if you change to the Thunderstore version, otherwise I'm not really sure why it happens, but I'll try to get it fixed in the next update

**Comment by FlagHater** [comment]
Yes, this is with the most recent (As of yesterday anyways) version of BoomboxPlaylists. I’ll try to see if it’s a comparability issue with one of my other mods. Thank you!
